src/pages/MyCars.js

```diff
@@ -0,0 +1,36 @@
+import styled from 'styled-components';
+import { useSelector } from 'react-redux';
+import { Grid } from '@mui/material';
+import { v4 as uuidv4 } from 'uuid';
+import CarCard from '../components/cars/CarCard';
+
+const GridContainer = styled(Grid)`
+  background-color: #f6f6f6;
+  padding: 2rem;
+  min-height: 100vh;
+  width: 100%;
+  max-width: 1320px;
+  margin: 0 !important;
+`;
+
+const MyCars = () => {
```

Comment:
- [ ] I couldn't find a way in the frontend to delete cars.  Since this is a [requirement](https://github.com/microverseinc/curriculum-final-capstone/blob/main/projects/business_requirements.md#project-requirements), kindly add a way to do so.

If the functionality already exists, kindly make it a little more obvious so your users are able to remove cars. 👍